app/views/companies/index.html.haml

```diff
@@ -0,0 +1,17 @@
+- if current_user.company.present?
+  %section.content-header
+    %h1 Company Detail
+  %section.content
+    .row
+      .col-md-6
+        .box.box-primary
+          .box-body
+            %p Company Name: #{current_user.company.name}
```

Comment:
duplicate hoping `current_user.company.*`. Can we use local variable for company and use it?